tcms/testruns/templates/testruns/get.html

```diff
@@ -66,8 +66,9 @@ <h2 class="card-pf-title" style="text-align: left">
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa-spinner"></span>{% trans 'Status' %}:
                     <input class="bootstrap-switch" id="status_button" type="checkbox"
-                        {% if not object.stop_date %}checked{% endif %}
-                        {% if not perms.testruns.change_testrun %}disabled{% endif %}>
+                    {% if not object.stop_date %}checked{% endif %}
+                        {% if not perms.testruns.change_testrun %}disabled{% endif %}
+                        {% if form.is_automated.value %}checked{% endif %} data-on-text="{% trans 'Running' %}" data-off-text="{% trans 'Finished' %}">
```

Comment:
We need the data-on-text/data-off-text attributes but not the extra if condition before them.